src/constants/index.ts

```diff
@@ -109,6 +109,7 @@ export const NS = {
   UI: {
     TimeField: `${nsBase}TimeField`,
     MultipleField: `${nsBase}Multiple`,
-    GroupField: `${nsBase}Group`
+    GroupField: `${nsBase}Group`,
+    Label: `${nsBase}label`
```

Comment:
Just to check: is it the intention that the capitalisation here is inconsistent with the vocabulary (but consistent with the other properties)? Or should this actually be:

```suggestion
    Label: `${nsBase}Label`
```

Reply:
The intention was to match the capitalization in the vocabulary. However - this is where the ui vocabulary is a bit confusing.  As best I can tell, there actually isn't a `label` property in the vocab. We have proposed one, as we think there's a definitive difference between an rdfs:label and a ui:label (one is explicitly intended for UI consumption and the other isn't). 

That being said, even the ui vocabulary itself uses `:label` and implies there is one in the local vocabulary (since that is the `:` prefix). I believe this is an omission that should be addressed in the ui vocab. 

But either way it would be ui:label and not Label :)

Reply:
That's not great :P

But if the vocab's capitalisation is to be respected, shouldn't it be

```suggestion
    label: `${nsBase}label`
```
?